pepper-apis/ddp-common/src/main/java/org/broadinstitute/ddp/constants/ConfigFile.java

```diff
@@ -202,17 +202,35 @@ public static class Auth0LogEvents {
-        public static final String REMOVAL_EXPIRE_UNIT = prefix + "removalExpireUnit";
-        public static final String REMOVAL_BATCH_SIZE = prefix + "removalBatchSize";
-        public static final String ENABLE_SCAN_RESULT_HANDLER = prefix + "enableScanResultHandler";
-        public static final String SCAN_RESULT_SUBSCRIPTION = prefix + "scanResultSubscription";
-        public static final String SIGNER_SERVICE_ACCOUNT = prefix + "signerServiceAccount";
+        public static final String ROOT_KEY = "fileUploads";
+
+        public static final String UPLOADS_BUCKET_KEY = "uploadsBucket";
+        public static final String SCANNED_BUCKET_KEY = "scannedBucket";
+        public static final String QUARANTINE_BUCKET_KEY = "quarantineBucket";
+        public static final String MAX_FILE_SIZE_BYTES_KEY = "maxFileSizeBytes";
+        public static final String MAX_SIGNED_URL_MINS_KEY = "maxSignedUrlMins";
+        public static final String REMOVAL_EXPIRE_TIME_KEY = "removalExpireTime";
+        public static final String REMOVAL_EXPIRE_UNIT_KEY = "removalExpireUnit";
+        public static final String REMOVAL_BATCH_SIZE_KEY = "removalBatchSize";
+        public static final String ENABLE_SCAN_RESULT_HANDLER_KEY = "enableScanResultHandler";
+        public static final String SCAN_RESULT_SUBSCRIPTION_KEY = "scanResultSubscription";
+        public static final String SIGNER_SERVICE_ACCOUNT_KEY = "signerServiceAccount";
+        public static final String UPLOADS_NOTIFICATION_TOPIC_KEY = "uploadsNotificationTopic";
+        public static final String UPLOADS_NOTIFICATION_SUBSCRIPTION_KEY = "uploadsNotificationSubscription";
+        
+        private static final String prefix = ROOT_KEY + ".";
+        public static final String UPLOADS_BUCKET = prefix + UPLOADS_BUCKET_KEY;
+        public static final String SCANNED_BUCKET = prefix + SCANNED_BUCKET_KEY;
+        public static final String QUARANTINE_BUCKET = prefix + QUARANTINE_BUCKET_KEY;
+        public static final String MAX_FILE_SIZE_BYTES = prefix + MAX_FILE_SIZE_BYTES_KEY;
+        public static final String MAX_SIGNED_URL_MINS = prefix + MAX_SIGNED_URL_MINS_KEY;
+        public static final String REMOVAL_EXPIRE_TIME = prefix + REMOVAL_EXPIRE_TIME_KEY;
+        public static final String REMOVAL_EXPIRE_UNIT = prefix + REMOVAL_EXPIRE_UNIT_KEY;
+        public static final String REMOVAL_BATCH_SIZE = prefix + REMOVAL_BATCH_SIZE_KEY;
+        public static final String ENABLE_SCAN_RESULT_HANDLER = prefix + ENABLE_SCAN_RESULT_HANDLER_KEY;
+        public static final String SCAN_RESULT_SUBSCRIPTION = prefix + SCAN_RESULT_SUBSCRIPTION_KEY;
+        public static final String UPLOADS_NOTIFICATION_TOPIC = prefix + UPLOADS_NOTIFICATION_TOPIC_KEY;
+        public static final String UPLOADS_NOTIFICATION_SUBSCRIPTION = prefix + UPLOADS_NOTIFICATION_SUBSCRIPTION_KEY;
+        public static final String SIGNER_SERVICE_ACCOUNT = prefix + SIGNER_SERVICE_ACCOUNT_KEY;
```

Comment:
In the app we are using an inconsistent mix of absolute and relative keypaths when accessing configuration values. The `_KEY` suffixed values were added to the above so that only the subconfig was required for the setup of the file upload notifications, and any required values from the global configuration were passed in via method parameters.

Reply:
No longer applicable.